lib/statsig_logging.ex

```diff
@@ -0,0 +1,123 @@
+defmodule Statsig.Logging do
+  use GenServer
+  require Logger
+
+  @type state :: %{
+    flush_interval: integer() | nil,
+    events: list(),
+    flush_timer: reference() | nil
+  }
+
+  def start_link(_) do
+    GenServer.start_link(__MODULE__, %{
+      flush_interval: 60_000,
+      events: [],
+      flush_timer: nil,
+    }, name: __MODULE__)
+  end
+
+  def initialize(flush_interval) do
+    GenServer.call(__MODULE__, {:initialize, flush_interval})
+  end
+
+  def shutdown do
```

Comment:
this doesn't actually shut down the genserver. Suggest calling this reset

Reply:
ive removed entirely - in logging i only really need a flush()